package.json

```diff
@@ -9,15 +9,18 @@
     "author": "Elementor Team",
     "homepage": "https://elementor.com/",
     "scripts": {
+        "setup-templates": "rm -rf ./templates && mkdir -p ./templates/playwright && mkdir -p ./templates/lighthouse && cp ./tests/playwright/templates/* ./templates/playwright && cp ./tests/lighthouse/templates/* ./templates/lighthouse",
+        "start-local-server": "npm run setup-templates && wp-env start --update",
         "test:php": "docker-compose run --rm wordpress_phpunit phpunit",
         "test:php_multisite": "docker-compose run -e WP_MULTISITE=1 --rm wordpress_phpunit phpunit",
         "test:qunit": "grunt qunit",
+        "test:setup:elements-regression": "wp-env run tests-cli \"bash -c 'bash elementor-config/setup.sh'\"",
+        "test:setup:playwright-sanity": "wp-env run cli \"wp --user=admin elementor library import-dir elementor-playwright-templates\" && wp-env run cli \"wp rewrite structure \"/%postname%/\" --hard\" && wp-env run cli \"wp theme activate twentytwentytwo\"",
         "test:playwright": "playwright test \"--config=tests/playwright/config/sanity.config.js\"",
-        "test:playwright:local": "playwright test \"--config=tests/playwright/config/local.config.js\"",
```

Comment:
Why delete this? How would we run local "gitignored" configs after this?

Reply:
https://github.com/elementor/elementor/pull/20453/commits/a6838f186f111325c14a2edc9b79f3b6e19a5f0e